data/themes/darktable-elegant-dark.css

```diff
@@ -37,7 +37,7 @@
 /* Modules box (plugins) */
 @define-color plugin_bg_color @grey_35;
 @define-color plugin_fg_color @grey_85;
```

Comment:
Could you change @grey_85 to @fg_color instead. It's not an issue on your PR but a thing I forgot in my last CSS PR as in darktable and grey files, as you can see, it's @fg_color that is applied. So just to be consistent between themes. Thanks!